server/odc-service/src/main/java/com/oceanbase/odc/service/integration/model/IntegrationProperties.java

```diff
@@ -53,13 +53,14 @@ public static class ApiProperties {
         private Body body;
         private boolean requestEncrypted = false;
         private boolean responseEncrypted = false;
+        private ResponseType responseType = ResponseType.JSON;
```

Comment:
it is not 'response type', but format of the response body.

similar concept may 'contentType'.

however, we may got content type from response header 'Content-Type', so if the content type is not defined,
ODC may detect from response header.

Reply:
I don't think we need to pay extra attention to the `Content-type` in response header. (Yes we can, but not necessary)

Apache http-core will parse response to `String` based on the Content-type automatically. So we should not care about different type of response content, just process the `String` according to the format specified by the integrator.

Reply:
Rename `responseType` to `responseContentType`